.github/workflows/build-test-package-python.yml

```diff
@@ -96,6 +96,26 @@ jobs:
         echo "Building for manylinux specialization ${MANYLINUX_VERSION} and target architecture ${TARGET_ARCH}"
         ./dockcross-manylinux-download-cache-and-build-module-wheels.sh cp${{ matrix.python-version }} $CMAKE_OPTIONS
 
+    - name: Validate build output
+      shell: bash
+      run: |
+        MANYLINUX_PLATFORM=${{ matrix.manylinux-platform }}
+        MANYLINUX_VERSION=`(echo ${MANYLINUX_PLATFORM} | cut -d '-' -f 1)`
+        TARGET_ARCH_NAME=`(echo ${MANYLINUX_PLATFORM} | cut -d '-' -f 2)`
+        if [[ ${TARGET_ARCH_NAME} == "x64" ]]; then
+          TARGET_ARCH_NAME="x86_64" # Match auditwheel naming convention
+        fi
+        WHEEL_PATTERN="dist/itk_*cp${{ matrix.python-version }}*manylinux${MANYLINUX_VERSION}*${TARGET_ARCH_NAME}.whl"
+
+        echo "Searching for wheels matching pattern ${WHEEL_PATTERN}"
+        dist_wheels=`(ls ${WHEEL_PATTERN})`
+        if [[ ${dist_wheels} ]]; then
```

Comment:
I am not familiar with this syntax -- this is checking that the files exist?

`twine check` may be useful here.

Reply:
Thanks, I was not aware of the [`twine check`](https://twine.readthedocs.io/en/stable/) utility, it does look like a good fit here.